src/include/transaction/timestamp_manager.h

```diff
@@ -95,6 +111,11 @@ class TimestampManager {
    */
   void RemoveTransactions(const std::vector<timestamp_t> &timestamps);
 
+  struct SpacedOutSpinLatch {
+    common::SpinLatch latch_;
+    byte padding_[common::Constants::CACHELINE_SIZE];
```

Comment:
Won't this cause memory fragmentation since each object is now larger than a cache line?